### DISCUSSION TITLE
4/29(토) 복습

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#115] [General]
제가 어제 실수로 과제를 냈내요 오늘은 복습을 해봅시다.!! 그리고 todoList는 잘되가나요? 제가 어제 말한 과제는 페메로 전송하겠습니다. 가능하면 둘다 화요일 안에 끝났으면 좋겠지만 일단 1순위는 투두입니다. 나보다 잘하니까 잘할꺼야~ 모르는 거 항상 물어보고~ ☺️

**Comment by Eunho0922** [comment]
넹
